topostats/grains.py

```diff
@@ -190,9 +190,14 @@ def find_grains(self):
         self.get_mask()
         self.tidy_border()
         self.label_regions(self.images["tidied_border"])
-        self.calc_minimum_grain_size(image=self.images["labelled_regions"])
-        self.remove_small_objects()
-        self.label_regions(self.images["objects_removed"])
-        self.get_region_properties()
-        self.colour_regions()
-        self.get_bounding_boxes()
+        try:
+            self.calc_minimum_grain_size(image=self.images["labelled_regions"])
+            self.remove_small_objects()
+            self.label_regions(self.images["objects_removed"])
+            self.get_region_properties()
+            self.colour_regions()
+            self.get_bounding_boxes()
+        # FIXME : Identify what exception is raised with images without grains and replace broad except
+        except Exception as e:
+            LOGGER.info(f"[{self.filename}] : No grains found.")
```

Comment:
I guess this is pythonic, but logging an exception as info seems odd.

Reply:
Its not that pythonic, if I had linted it I would have been told the exception is too broad.  But this is just a place holder for now, I've requested images that have no grains and will see what exceptions are actually raised and handle them appropriately (mostly it will be a pass but I figured logging some information to say why certain steps aren't carried out would be informative).